### PR TITLE
Fix export datavolume tests duplicate mac

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:
If macpool installed the exported VM definition contains the mac address of the source VM, this causes the test to fail when trying to replicate into the same cluster.

Clearing out the mac before attempting to create a new one fixes the problem

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/storage/export.go [hunk 1]
```diff
@@ -1271,12 +1271,10 @@ var _ = SIGDescribe("Export", func() {
 		Expect(*snapshot.Status.SourceUID).To(Equal(vm.UID))
 
 		contentName := *snapshot.Status.VirtualMachineSnapshotContentName
-		Expect(snapshot.Status.Indications).To(BeEmpty())
```
Comment (akalenyu) [inline tests/storage/export.go:1274]: hmm whats wrong with these?

File: tests/storage/export.go [hunk 2]
```diff
@@ -1271,12 +1271,10 @@ var _ = SIGDescribe("Export", func() {
...
 		content, err := virtClient.VirtualMachineSnapshotContent(vm.Namespace).Get(context.Background(), contentName, metav1.GetOptions{})
 		Expect(err).ToNot(HaveOccurred())
 
 		Expect(*content.Spec.VirtualMachineSnapshotName).To(Equal(snapshot.Name))
-		Expect(content.Spec.Source.VirtualMachine.Spec).To(Equal(vm.Spec))
```
Comment (akalenyu) [inline tests/storage/export.go:1279]: same as above

Comment (awels) [reply tests/storage/export.go:1279]: Turns out to also not always be to be equal, again not really that import that they match exactly for this test.

File: tests/storage/export.go (hunk 1 above)
Comment (mhenriks) [reply tests/storage/export.go:1274]: If you are using image with guest agent, it should not be empty?  Do some tests have guest agent and some do not?  Maybe that can be a flag?

Comment (mhenriks) [reply tests/storage/export.go:1274]: But this aspect is not particularly relevant for this test suite

Comment (akalenyu) [reply tests/storage/export.go:1274]: We will keep pursuing this outside the PR

**Pull request merged**